kitsune/wiki/tests/test_templates.py

```diff
@@ -1013,6 +1013,143 @@ def test_keywords_require_permission(self):
         new_rev = Revision.objects.filter(document=doc).order_by('-id')[0]
         eq_('keyword1 keyword2', new_rev.keywords)
 
+    def test_restore_draft_revision(self):
+        rev = ApprovedRevisionFactory(is_ready_for_localization=True)
+        doc = rev.document
+        doc.is_localizable = True
+        doc.save()
```

Comment:
You could do this more easily with `rev = ApprovedRevisionFactory(is_ready_for_localization=True, document__is_localizable=True)`.